sample/jvm-app/src/jvmMain/kotlin/com/icerockdev/desktop/Main.kt

```diff
@@ -4,29 +4,33 @@
 
 package com.icerockdev.desktop
 
-import androidx.compose.desktop.Window
 import androidx.compose.foundation.Image
-import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.layout.size
 import androidx.compose.material.Surface
 import androidx.compose.material.Text
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
-import androidx.compose.ui.unit.IntSize
+import androidx.compose.ui.graphics.toComposeImageBitmap
+import androidx.compose.ui.unit.DpSize
 import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.WindowState
+import androidx.compose.ui.window.singleWindowApplication
 import com.icerockdev.library.MR
 import com.icerockdev.library.Testing
-import dev.icerock.moko.resources.compose.toImageBitmap
 
 fun main() {
     val testing = Testing
-    Window("moko-resources", IntSize(1080, 960)) {
+    singleWindowApplication(
+        title = "moko-resources",
+        state = WindowState(size = DpSize(1080.dp, 960.dp))
+    ) {
         Surface(Modifier.fillMaxSize()) {
             Column {
                 Image(
-                    bitmap = testing.getDrawable().image.toImageBitmap(),
+                    bitmap = testing.getDrawable().image.toComposeImageBitmap(),
```

Comment:
we can convert `java.awt.image.BufferedImage` to compose by this? so we should add to `toImageBitmap` extension Deprecated annotation with suggestion change it to `toComposeImageBitmap`